conf/etc/nginx/sites-available/default

```diff
@@ -23,6 +23,7 @@ server {
 
 	root /opt/piecewise_web;
 	index index.html index.htm;
+	sendfile off;
```

Comment:
Out of curiosity, what does this change aim to fix? I couldn't quickly find which commit  this change came from.